packages/core/src/Stores/notification-store.js

```diff
@@ -1640,4 +1649,50 @@ export default class NotificationStore extends BaseStore {
             platform: 'Account',
         });
     };
+
+    handleAUDCurrencyRemovalNotification = aud_account_closure_status => {
```

Comment:
This code snippet is tied to the logic of the task, it has localize and UI components which is not supposed to be in a store, how about implementing this as a hook?

Reply:
I would agree with you @matin-deriv, but we set all our notifications in this store. basically all the logic for notifications is here, and the implementation is the same as for other cases. the hook is not needed here I believe :)